static/app/views/settings/organizationTeams/teamMembers.spec.jsx

```diff
@@ -65,11 +65,7 @@ describe('TeamMembers', function () {
       />
     );
 
-    await userEvent.click(
-      (
-        await screen.findAllByRole('button', {name: 'Add Member'})
-      )[0]
-    );
+    await userEvent.click((await screen.getAllByTestId('add-member'))[0]);
```

Comment:
@leedongwei I just published this for discussion. This change would help fix the broken test issue but I don't understand why the button cannot be found using ByRole search after my changes 🤔